hexa/databases/api.py

```diff
@@ -0,0 +1,97 @@
+import re
+
+import psycopg2
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from psycopg2 import sql
+from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
+
+
+def get_db_server_credentials():
+    return {
+        "role": settings.WORKSPACE_DATABASE_ROLE,
+        "password": settings.WORKSPACE_DATABASE_PASSWORD,
+        "host": settings.WORKSPACE_DATABASE_HOST,
+        "port": settings.WORKSPACE_DATABASE_PORT,
+    }
+
+
+def format_db_name(name: str):
+    db_name = ""
+    for char in name.lower().replace(" ", "_"):
```

Comment:
Since you take the slug of the workspace, there is already no space in it. It makes more sense to replace `-` by `_` to keep the words

Reply:
So you advice to not replace or change the replacement stirng?